qpmodel/LogicCard.cs

```diff
@@ -183,9 +183,9 @@ public override ulong LogicJoinCE(LogicJoin node)
                     mindlr = 0;
```

Comment:
> When histogram is not present, force mcv estimation to work on values not in mcv.
In PG, mcv is a complementary to histogram, meaning mcv contains values not presented in histgram. Are you changing to that route?